src/main/java/de/teamlapen/vampirism/entity/converted/EntityConvertedCreature.java

```diff
@@ -258,12 +259,12 @@ private void writeOldEntityToNBT(NBTTagCompound nbt) {
         if (!nil()) {
             try {
                 NBTTagCompound entity = new NBTTagCompound();
-                entityCreature.isDead = false;
+                entityCreature.dead = false;//TODO is protected
```

Comment:
Careful. This is a completely different variable than the old one. (Different class, different effect)
The original Entity#isDead is now inverted and renamed to Entity#isAlive (or respectively removed).
The only intention is to trick the check in Entity#writeUnlessPassenger